tests/e2e/performance/test_pvc_creation_deletion_performance.py

```diff
@@ -0,0 +1,296 @@
+        return pod_obj
+
+    @pytest.mark.usefixtures(base_setup.__name__)
+    def test_multiple_pvc_deletion_measurement_performance(self, teardown_factory):
+        """
+        Measuring PVC deletion time of 120 PVCs in 180 seconds
+
+        Args:
+            teardown_factory: A fixture used when we want a new resource that was created during the tests
+                               to be removed in the teardown phase.
+        Returns:
+
+        """
+        number_of_pvcs = 120
+        pvc_size = "1Gi"
+        msg_prefix = f"Interface: {self.interface}, PVC size: {pvc_size}."
+
+        log.info(f"{msg_prefix} Start creating new 120 PVCs")
+
+        pvc_objs = helpers.create_multiple_pvcs(
+            sc_name=self.sc_obj.name,
+            namespace=defaults.ROOK_CLUSTER_NAMESPACE,
+            number_of_pvc=number_of_pvcs,
+            size=pvc_size,
+            burst=True,
+        )
+
+        for pvc_obj in pvc_objs:
+            pvc_obj.reload()
+            teardown_factory(pvc_obj)
+        with ThreadPoolExecutor(max_workers=5) as executor:
+            for pvc_obj in pvc_objs:
+                executor.submit(
+                    helpers.wait_for_resource_state, pvc_obj, constants.STATUS_BOUND
+                )
+
+                executor.submit(pvc_obj.reload)
+
+        pod_objs = []
+        for pvc_obj in pvc_objs:
+            pod_obj = self.write_file_on_pvc(pvc_obj, 0.3)
+            pod_objs.append(pod_obj)
+
+        # Get pvc_name, require pvc_name to fetch deletion time data from log
+        threads = list()
+        for pvc_obj in pvc_objs:
+            process = threading.Thread(target=pvc_obj.reload)
+            process.start()
+            threads.append(process)
+        for process in threads:
+            process.join()
+
+        pvc_name_list, pv_name_list = ([] for i in range(2))
+        threads = list()
+        for pvc_obj in pvc_objs:
+            process1 = threading.Thread(target=pvc_name_list.append(pvc_obj.name))
+            process2 = threading.Thread(target=pv_name_list.append(pvc_obj.backed_pv))
+            process1.start()
+            process2.start()
+            threads.append(process1)
+            threads.append(process2)
+        for process in threads:
+            process.join()
+        log.info(f"{msg_prefix} Preparing to delete 120 PVC")
+
+        # Delete PVC
+        for pvc_obj, pod_obj in zip(pvc_objs, pod_objs):
+            pod_obj.delete(wait=True)
+            pvc_obj.delete()
+            pvc_obj.ocp.wait_for_delete(pvc_obj.name)
+
+        # Get PVC deletion time
+        pvc_deletion_time = helpers.measure_pv_deletion_time_bulk(
+            interface=self.interface, pv_name_list=pv_name_list
+        )
+        log.info(
+            f"{msg_prefix} {number_of_pvcs} bulk deletion time is {pvc_deletion_time}"
+        )
+
+        # accepted deletion time is 2 secs for each PVC
+        accepted_pvc_deletion_time = number_of_pvcs * 2
```

Comment:
what is the bulk deletion time we are observing?
since we run deletion in parallel, deletion should take much lesser time. for eg., 120 PVC deletion in your case would be 240 seconds but I guess it should get completed in less than 120 seconds.

Reply:
@keesturam  actually the deletion is serial in this case, not parralel - Avi and I had this conversation as well last week. 
The plan is to handle ( if it's needed) parallel deletion in one of the future PRs, since currently we don't have a function that deletes mltiple PVCs as bulk.

Reply:
okay, maybe raise a github issue to track this requirement

Reply:
https://github.com/red-hat-storage/ocs-ci/issues/4097   - I've opened a git hub issue and put a "performance" label there.